### PR TITLE
Feature: Added "Properties" button to the Details Pane

### LINKED ISSUE #12796: Feature: Add "Edit properties" button to the Details pane
### What feature or improvement do you think would benefit Files?

Users can already open properties from the toolbar but we can make it easier to access if we display a labeled button in the Details pane.

### Requirements

- This proposal will add an "Edit properties" button to the bottom of the Details pane.

### Files Version

v2.5.11

### Windows Version

Windows 11

### Comments

_No response_

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #12796 

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open the details pane
   2. Click on `Properties`

**Screenshots (optional)**
image

### CODE REVIEW

**Comment by Lukiluc29** [comment]
Hello, I'm sorry for intruding into your commit, but I would like to know if it would be possible to move the button to the right instead of centering it in the preview pane?
Capture d'écran 2023-07-10 111652

**Comment by ferrariofilippo** [comment]
> Hello, I'm sorry for intruding into your commit, but I would like to know if it would be possible to move the button to the right instead of centering it in the preview pane?

I'd like to get more feedback before changing it

**Comment by yaira2** [comment]
The proposed design has it on the left, this way it lines up with the other details.

File: src/Files.App/UserControls/Pane/PreviewPane.xaml [hunk 1]
```diff
@@ -267,6 +268,20 @@
 						</DataTemplate>
 					</ItemsControl.ItemTemplate>
 				</ItemsControl>
+				<Button
```
Comment (yaira2) [inline src/Files.App/UserControls/Pane/PreviewPane.xaml:271]:
```suggestion
				<!-- Open Properties Button -->
				<Button
```

File: src/Files.App/UserControls/Pane/PreviewPane.xaml [hunk 2]
```diff
@@ -267,6 +268,20 @@
...
+					x:Name="DetailsOpenProperties"
+					Margin="0,0,0,8"
+					HorizontalAlignment="Center"
```
Comment (yaira2) [inline src/Files.App/UserControls/Pane/PreviewPane.xaml:274]:
```suggestion
					HorizontalAlignment="Left"
```

**Review by yaira2 — APPROVED** [review]
LGTM

**Comment by mdtauk** [comment]
Can we change the data for "Horizontal Resolution" and "Vertical Resolution" to be just 
```
Resolution:
1920 × 1080 px

**Comment by yaira2** [comment]
Agreed, let's do that

**Comment by ferrariofilippo** [comment]
> Can we change the data for "Horizontal Resolution" and "Vertical Resolution" to be just
> 
> ```
> Resolution:
> 1920 × 1080 px
> ```
> 
> EDIT: Please note the use of the Multiplication symbol **×**, and not the lowercase **x**

Actually, `Horizontal Resolution` and `Vertical Resolution` are not the "Dimension" of the image (like 1920 x 1080 px) but rather pixels/inch

image

**Pull request merged**